experiments/mp-game-6/draw.creatures.js

```diff
@@ -116,20 +161,46 @@ function drawTestCreature(stim, creatureInd, scale, roundProps){
             markAsSpecies(id);
             roundProps.selected_test_stim.push(id);
         }
-        console.log(roundProps.selected_test_stim)
     });
 }
   
 function darken(id) {
-    $(id).css({"opacity": 1.0});
+    $(id).css({"opacity":1.0});
+}
+function lighten(id){
+    $(id).css({"opacity":.5});
 }
   
-function showSpeciesIndicator(id, speciesName) {
-    var labelID = id + "_label";
-    $(labelID).css("visibility", "visible");
-  
-    if ($(id).attr("belongs_to_concept") === "true")
-        $(id).css({"border":"2px dashed black"});
+function turnLightsOn(buttonsOn, test1, numButtons, numLights, lightsPrev) {
+    var lightsOn = [];
+    var arrOfButtons = [];
+    for(var i = 1; i<=numButtons; i++){
+        if(buttonsOn.includes("#button" + i)) arrOfButtons.push(true);
+        else arrOfButtons.push(false);
+    }
+    for(var j = 1; j<=numLights; j++){
+        $("#light" + j).css({"opacity":0.5});
+    }
+    var setting1 = new Setting(arrOfButtons)
```

Comment:
As above, `Setting` seems a bit hardcoded now (left button, right button), it's likely you can just get rid of this class and use the underlying fixed-length array as input to `test1`